.eslintignore

```diff
@@ -4,7 +4,6 @@ packages/node_modules/generator-ciscospark/generators/plugin/templates/src/_inde
 
 # The follwowing is copied from .gitignore
 .DS_Store
-/node_modules
```

Comment:
Just curious to know why do we have to include node_modules for eslint

Reply:
after eslint upgrade by default it was doing it so removed it , Did not throw any error to me